esp/bindings/http/platform/httpbinding.hpp

```diff
@@ -461,8 +461,18 @@ class esp_http_decl EspHttpBinding :
                             const char *serviceName, const char* methodName);
     void sortResponse(IEspContext& context, CHttpRequest* request,MemoryBuffer& contentconst,
                             const char *serviceName, const char* methodName);
+
+    enum SchemaType {
```

Comment:
I'm not a huge fan of the internal function having to learn about the name and requirements of the external use case.  I'd rather see flags where the caller can specify what features it needs.

1. wsdl, which is a different output format for the function.
2. Add annotations, which adds additional meta data to the output.
etc.

In the end most of these use cases just need the schema to know the basic structure of the request.  The use case name shouldn't be exposed to the internals.